packages/image_picker/lib/image_picker.dart

```diff
@@ -32,14 +32,17 @@ class ImagePicker {
   /// If specified, the image will be at most [maxWidth] wide and
   /// [maxHeight] tall. Otherwise the image will be returned at it's
   /// original width and height.
+  /// The [imageQuality] argument resize the image as per the input, ranging from 0-100 compression.
+  /// 0 meaning compress for small size, 100 meaning compress for max quality.
+  /// Some formats, like PNG which is loss less, will ignore the quality setting
```

Comment:
Seems only apply to iOS? It seems Android can compress PNGs.

Reply:
in android, for PNG(lossless formats) it will ignore the compression

Reply:
ok, so you are saying Android behaves the same as iOS. Then we can also print a message from Android if user picked a PNG but the quality is set to lower than 100. I think it is a good info for developer when debugging or profiling.

Reply:
yes, added a log in the Android part, with the same log as in iOS